docs/readthedocs/source/doc/LLM/Overview/KeyFeatures/optimize_model.md

```diff
@@ -18,10 +20,46 @@ from bigdl.llm import optimize_model
 model = optimize_model(model)
 ```
 
+```eval_rst
+.. note::
```

Comment:
maybe add a new section "more precisions" for IN3/INT8, etc. instead of put it in Note. Sometime impatient readers just ignore those notes.

Reply:
> and maybe add some explanations about the difference of sym and asym

Have added descriptions according to above comments.